### PR TITLE
#441: Implemented simple macro for simplification of file creation

### PR DESCRIPTION
**Pull request by kelebra**
Implemented a simple macro as suggested by @dlwh
So now we can call it like this:
`csvread(file"src/test/resources/cogp/cogp_no_missing_points.csv")`

### CODE REVIEW

File: macros/src/main/scala/breeze/macros/util.scala [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package breeze.macros
+
+import java.io.File
+
+object util {
+
+  implicit class FileMacro(val sc: StringContext) extends AnyVal {
+
+    def file(args: Any*): File = new File(sc.s(args: _*))
```
Comment (dlwh) [inline macros/src/main/scala/breeze/macros/util.scala]: this is great! I don't think this isn't really a macro though right?

Could we maybe just put this in, say, a new breeze.util package object?

Comment (kelebra) [inline macros/src/main/scala/breeze/macros/util.scala]: Totally agree, will commit today. Do you want me to squash commits afterwards?

**Comment by kelebra** [comment]
Please let me know if anything else need to be done

**Comment by dlwh** [comment]
thanks!

**Pull request merged**